server/logs.js

```diff
@@ -84,7 +84,8 @@ if (!util.isUndefined(watsonServices)) {
   const watsonCredentials = watsonServices[0].credentials;
 
   const watsonOptions = {
-    url: 'https://gateway.watsonplatform.net/tone-analyzer/api/',
+    // For the url to be region-agnostic
+    url: watsonCredentials.url,
     username: watsonCredentials.username,
     password: watsonCredentials.password,
     version_date: '2016-05-19',
```

Comment:
is there a most recent version to use?

Reply:
should be ok, there is a 2017er version but with breaking changes

Reply:
tracking it for the future https://github.com/IBM-Cloud/insurance-bot-dashboard/issues/66